application/lib/templates.dart

```diff
@@ -10,20 +10,20 @@ class AtomDescription {
     this.properties,
   });
 
-  final String identifier;
-  final String className;
-  final Map<String, PropertyValue> properties;
+  final String? identifier;
+  final String? className;
+  final Map<String, PropertyValue>? properties;
```

Comment:
the rest of the code seems to imply these are non-nullable

Reply:
fixed